src/redux/books/books.js

```diff
@@ -1,30 +1,77 @@
-import bookList from '../../FakeBookData';
+import booksURL from '../../apiURL';
+
 // Action
 const ADD_BOOK = 'ADD_BOOK';
 const DELETE_BOOK = 'DELETE_BOOK';
+const GET_BOOKLIST = 'GET_BOOKLIST';
 
 // Action creators
+// Add book
+export const addBook = (book) => async (dispatch) => {
+  const response = await fetch(booksURL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: book.item_id,
+      title: book.title,
+      author: book.author,
+      category: 'Action',
+    }),
+  });
+  if (response.ok) {
+    dispatch({ type: ADD_BOOK, book });
+  }
+};
```

Comment:
Please be sure to use createAsyncThunk in your action creators so that your project meets the requirements 🚀